runtime/reconcilers/metrics_view.go

```diff
@@ -152,6 +153,36 @@ func (r *MetricsViewReconciler) validate(ctx context.Context, mv *runtimev1.Metr
 	return errors.Join(errs...)
 }
 
+func validateDimension(
+	ctx context.Context,
+	olap drivers.OLAPStore,
+	t *drivers.Table,
+	d *runtimev1.MetricsViewSpec_DimensionV2,
+	fields map[string]*runtimev1.StructType_Field,
+) error {
+	if d.Expression == "" {
+		if d.Column == "" {
+			return nil
+		}
+		if _, isColumn := fields[d.Column]; !isColumn {
+			return fmt.Errorf("column not found: %s", d.Column)
+		}
+		return nil
+	}
+
+	err := olap.Exec(ctx, &drivers.Statement{
+		Query:  fmt.Sprintf("SELECT %s from %s GROUP BY 1", d.Expression, safeSQLName(t.Name)),
```

Comment:
1. Is the `GROUP BY 1` approach supported by Druid?
2. nit: uppercase `FROM`

Reply:
Group by 1 is working for druid, aggregates throw error as. expected.